travel-journal/index.md

```diff
@@ -0,0 +1,910 @@
+- Some knowledge of the Vue.js Composition API.
+
+## Creating A Vue.js Project
+
+Open up your command prompt or terminal app and run the following command:
+
+```bash
+npm create vue@3
+```
+
+During project creation, when prompted with questions, choose the following configurations:
+
+```bash
+✔ Project name: … ./directus-travel-journal
+✔ Package name: … -directus-travel-journal
+✔ Add TypeScript? … No / Yes
+✔ Add JSX Support? … No / Yes
+✔ Add Vue Router for Single Page Application development? … No / Yes
+✔ Add Pinia for state management? … No / Yes
+✔ Add Vitest for Unit Testing? … No / Yes
+✔ Add an End-to-End Testing Solution? › No
+✔ Add ESLint for code quality? … No / Yes
+```
+
+Once completed, run the following command to install dependencies and start the development server:
+
+```bash
+cd directus-travel-journal
+code .
+npm install
+npm run dev
+```
+
+Once everything is set up, visit the following address  - localhost:5173 **and you should see the vite welcome page:
+
+![Viewing the vite welcome page](./images/1.png)
+
+## Creating a Journal and Users Collection
+
+In your Directus project, navigate to Settings -> Data Model and create a new collection called `journal`. Use `journal` as the table name and leave every other configuration as their defaults.
+
+Create the following fields with the following types:
+
+- `title`: Input - string (*required*).
+- `description`: Textarea - text (*required*).
+- `photo`: Image - uuid (*required*).
+- `city`: Input - string (*required*).
+- `country`: Input - string (*required*).
+
+Add a description of the `journal` collection and proceed to creating the collection.
+
+![Creating the journal collection](./images/2.png)
+
+Head over to Settings→Access Control, create customer role with only App Access. Give it create, read, update and delete access to the `journal` collection, add a description and then proceed to creating the customer role while taking note of the customer role id:
+
+![Creating a customer role](./images/3.png)
+
+After creating the customer role, you will be directed to the Access Control page. Click on the Public role and enable create, read, update and delete access to the `journal` collections and also for `directus_files` under system collections:
+
+![Authorizing public role](./images/4.png)
+
+## Preparing the Vue.js Project
+
+In your Vue.js project, navigate to the `src` folder. Delete the `base.css` file and edit the `main.css` file to contain the following: 
+
+```css
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+
+*, body{
+  font-family: 'Poppins', sans-serif;
+}
+```
+
+Doing this will set the overall font for the Vue.js application to [Poppins](https://fonts.google.com/specimen/Poppins).
```

Comment:
Because I don't want you to have to redo lots of screenshots, let's leave this as-is. But I'll share that I tend to not style any pages in technical tutorials as it is both subjective/often customized by readers, and a distraction from the core knowledge we are trying to impart. 

This is more just a note to help your technical writing in future.